src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let off = 0;
```

Comment:
Let's create more meaningful name (offCost or discount)
```suggestion
  let offCost = 0;
```